src/index.js

```diff
@@ -1,41 +1,19 @@
 import './style.css';
+import { addList, renderTasks } from './app.js';
```

Comment:
- [ ] I think it would be great here if you fix this problem as well because after deleting all the tasks I can see the last task without any coressponding options to delete or update like this image ⬇ but I could see it after rereshing the page so please fix this issue as well. 🤩🚀💯🔥 

![image](https://user-images.githubusercontent.com/58553711/229015593-40e7abc4-f9f6-43ba-9218-efac6fea1c4e.png)